ci/authn-azure/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+AWS_PEM_FILE_LOCATION="<location of pem file to login to machine>"
+USERNAME="<username of AWS machine>"
+AWS_MACHINE="<AWS machine>"
+
+pushd ../..
+  . version_utils.sh
```

Comment:
why does this clock need to be indented

Reply:
we indent blocks inside `pushd` so it's clear it runs in a different place.